src/js/rx/components/ConfirmRefillModal.jsx

```diff
@@ -39,8 +39,11 @@ class ConfirmRefillModal extends React.Component {
                   <span className="rx-modal-drug">{this.props.drug}</span>
                   <span className="rx-modal-dosage"> {this.props.dosage}</span>
                 </div>
+                <div className="rx-modal-facility">
+                  Facility name: {this.props.facilityName}
+                </div>
                 <div className="rx-modal-lastrefilled">
-                  Last refilled: {this.props.lastRefilled}
+                  Last requested: {this.props.lastRefilled}
```

Comment:
I think this is back to using `orderedDate`, according to https://github.com/department-of-veterans-affairs/kudos-team/blob/master/Prescriptions/design/Rx%20statuses.md.